geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/CommandManager.java

```diff
@@ -80,16 +88,17 @@ public CommandManager() {
    * this is used when getting the instance in a cache server. We are getting the
    * user-command-package from distribution properties. used by OnlineCommandProcessor.
    */
-  public CommandManager(final Properties cacheProperties, InternalCache cache) {
-    if (cacheProperties != null) {
-      this.cacheProperties = cacheProperties;
+  public CommandManager(final Properties newCacheProperties, InternalCache cache) {
+    if (newCacheProperties != null) {
+      this.cacheProperties.putAll(newCacheProperties);
```

Comment:
If you are planning to do code cleanup; reference to "this" can be removed in many places :)

Reply:
the removal of the keyword 'this' does not make code any cleaner... in actual fact, it makes it harder to read, as we are spoilt with IDE's that indicate fields and properties in different colors. As for reading this as text, makes it evident that we are referring to the field defined on the CommandManager.